lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -559,14 +562,32 @@ class ConvertAtenConv2dOp : public OpConversionPattern<AtenConv2dOp> {
 
     auto stridesAttr = rewriter.getI64VectorAttr(strideInts);
     auto dilationAttr = rewriter.getI64VectorAttr(dilationInts);
-    Value conv2d =
+    Value conv;
```

Comment:
can you remove the commented out code and add a // TODO:  add support for 2d and 2d case